docs/data/date-pickers/date-picker/SubComponentsCalendars.tsx

```diff
@@ -1,39 +1,24 @@
 import * as React from 'react';
-import dayjs, { Dayjs } from 'dayjs';
+import dayjs from 'dayjs';
 import Grid from '@mui/material/Grid';
 import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
 import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
 import { DateCalendar } from '@mui/x-date-pickers/DateCalendar';
 import { MonthCalendar } from '@mui/x-date-pickers/MonthCalendar';
 import { YearCalendar } from '@mui/x-date-pickers/YearCalendar';
 
-const minDate = dayjs('2020-01-01T00:00:00.000');
-const maxDate = dayjs('2034-01-01T00:00:00.000');
-
 export default function SubComponentsCalendars() {
-  const [value, setValue] = React.useState<Dayjs | null>(dayjs('2022-04-07'));
-
   return (
     <LocalizationProvider dateAdapter={AdapterDayjs}>
       <Grid container spacing={3}>
         <Grid item xs={12} md={6}>
-          <DateCalendar value={value} onChange={(newValue) => setValue(newValue)} />
+          <DateCalendar defaultValue={dayjs('2022-04-07')} />
         </Grid>
         <Grid item xs={12} md={6}>
-          <MonthCalendar
-            value={value}
-            minDate={minDate}
-            maxDate={maxDate}
-            onChange={(newValue) => setValue(newValue)}
-          />
+          <MonthCalendar defaultValue={dayjs('2022-04-07')} />
         </Grid>
         <Grid item xs={12} md={6}>
-          <YearCalendar
-            value={value}
-            minDate={minDate}
-            maxDate={maxDate}
-            onChange={(newValue) => setValue(newValue)}
-          />
+          <YearCalendar defaultValue={dayjs('2022-04-07')} />
```

Comment:
> but also not having the min and max dates on this calendar causes the calendar to render A LOT of years and also shows a bug—the view is not scrolled into the selected year

For me the minDate / maxDate were defined here because in the past they were mandatory (see #4814)
But if we think that they are useful in the demo, I can of course add them back.

For the bug, I thought @alexfauquette fixed it some time ago, but maybe not in this situation.

Reply:
Yes, it sounds like a bug in the computation of the scroll, because the scroll is done (scrollbar is not at the top) but a bit bellow the expected time (2022) Need to investigate in another PR

![image](https://user-images.githubusercontent.com/45398769/203264893-c97ef524-3c7b-459f-9f98-42a32a39bb7b.png)